client/dashboard/domains/domain-overview/settings.tsx

```diff
@@ -0,0 +1,31 @@
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { __experimentalVStack as VStack } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { domainQuery } from '../../app/queries/domain';
+import { domainRoute } from '../../app/router/domains';
+import { SectionHeader } from '../../components/section-header';
+import { SummaryButtonList } from '../../components/summary-button-list';
+import DomainContactDetailsSettingsSummary from '../domain-contact-details/summary';
+import DnsSettingsSummary from '../domain-dns/summary';
+import DomainForwardingsSettingsSummary from '../domain-forwardings/summary';
+import DomainSecuritySettingsSummary from '../domain-security/summary';
+import NameServersSettingsSummary from '../name-servers/summary';
+import DomainGlueRecordsSettingsSummary from '../overview-glue-records/summary';
+
+export default function DomainOverviewSettings() {
+	const { domainName } = domainRoute.useParams();
+	const { data: domain } = useSuspenseQuery( domainQuery( domainName ) );
+	return (
+		<VStack spacing={ 3 }>
+			<SectionHeader title={ __( 'Settings' ) } level={ 3 } />
+			<SummaryButtonList>
+				<DnsSettingsSummary domain={ domain } />
+				<NameServersSettingsSummary domain={ domain } />
```

Comment:
In the newest Figma, we see "Name servers" as the first item. Can we update to reflect that?

<img width="720" height="858" alt="image" src="https://github.com/user-attachments/assets/1239b789-4595-4f9d-833d-9d84c5f23b87" />